repair_shards.h

```diff
@@ -20,11 +20,11 @@
 /* templates for SQL commands used during shard placement repair */
 #define DROP_REGULAR_TABLE_COMMAND "DROP TABLE IF EXISTS %s"
 #define DROP_FOREIGN_TABLE_COMMAND "DROP FOREIGN TABLE IF EXISTS %s"
+#define COPY_RELATION_QUERY "SELECT worker_copy_shard_placement(%s, %s, %d)"
```

Comment:
A few ``#define`` name alternatives to make the name similar to Citus (we use TABLE_FETCH_COMMAND in Citus). What do you think about the following alternatives?

1. COPY_PLACEMENT_COMMAND or COPY_SHARD_PLACEMENT_COMMAND
2. COPY_PLACEMENT_QUERY or COPY_SHARD_PLACEMENT_QUERY